app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize user
+    if user.admin?
+      can :manage, :all
+    else
+      can %i(read update), Order, user_id: user.id
+      can :create, Order
```

Comment:
còn nhiều model nữa mà e ei
e có cập nhập user k?

Reply:
Vâng e thêm rồi ạ

Reply:
ở các controller k cần load and authorize à e

Reply:
Không ạ, vì ở các route đó e k cần chặn ngưởi dùng ở bất kỳ role nào ạ, ví dụ như ở controller room hay categories thì e chỉ có các hàm như index hay show, không có các hàm liên quan đến quyền cá nhân hay quản trị nên e k thêm vào ạ